src/couch_replicator_manager.erl

```diff
@@ -220,6 +240,12 @@ handle_call({rep_db_checkpoint, DbName, EndSeq}, _From, State) ->
     true = ets:insert(?DB_TO_SEQ, Entry),
     {reply, ok, State};
 
+% Ignore checkpoints from previous epoch.
+handle_call({rep_db_checkpoint, DbName, EndSeq, Epoch}, _From, State) ->
+    Msg = "~p : ignoring checkpoint ~p for ~p, epoch doesn't match ~p =/= ~p",
+    twig:log(notice, Msg, [?MODULE, EndSeq, DbName, Epoch, State#state.epoch]),
```

Comment:
I'd make this a debug message or remove it entirely. We know it happens and we're not doing anything to make it go away.